tsconfig.json

```diff
@@ -52,6 +52,8 @@
     "src/components/VNotificationBanner.vue",
     "src/components/VMigrationNotice.vue",
     "src/components/VVariations.vue",
+    "src/components/VInputField/VInputField.vue",
+    "src/components/VHeader/VSearchBar/VSearchBar.vue",
```

Comment:
I did not add `VSearchButton` here because it depends on a couple of composables that are being converted in other PRs.